product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx

```diff
@@ -0,0 +1,139 @@
+
+1.  Select the project where you created the database cluster.
+
+1.  Within your project, select **Migrate** > **Credentials**.
+
+1.  Unzip the credentials folder and copy it to the host where the reader is installed.
+
+## Configuring the reader
+
+Set the following environment variables in `/opt/cdcreader/run-cdcreader.sh` with the right values:
+
+```shell
+### set the following environment variables:
+
+##########################################
+# Transporter Cloud Configuration        #
+##########################################
+
+# This ID is used to identify the cdcreader.
+#export DBZ_ID=
+
+# Now we only support aws
+#export CLOUD_PROVIDER=
+
+# No need to change about this field
+#export RW_SERVICE_HOST=https://transporter-rw-service.biganimal.com
+
+# You need to create migration credentials in EDB postgresAI platform and set these fields with the path of credential files
+#export TLS_PRIVATE_KEY_PATH=$MY_CREDENTIALS_PATH/client-key.pem
+#export TLS_CERTIFICATE_PATH=$MY_CREDENTIALS_PATH/client-cert.pem
+#export TLS_CA_PATH=$MY_CREDENTIALS_PATH/int.crt
+#export APICURIOREQUEST_CLIENT_KEYSTORE_LOCATION=$MY_CREDENTIALS_PATH/client.keystore.p12
+#export APICURIOREQUEST_TRUSTSTORE_LOCATION=$MY_CREDENTIALS_PATH/int.truststore.p12
+#export KAFKASECURITY_CLIENT_KEYSTORE_LOCATION=$MY_CREDENTIALS_PATH/client.keystore.p12
+#export KAFKASECURITY_TRUSTSTORE_LOCATION=$MY_CREDENTIALS_PATH/int.truststore.p12
+
+##########################################
+# Transporter Source DB Configuration    #
+##########################################
+
+# A sample configuration to create a single postgres database connection:
+#export DBZ_DATABASES_0__TYPE=POSTGRES
+#export DBZ_DATABASES_0__HOSTNAME=localhost
+#export DBZ_DATABASES_0__PORT=5432
+#export DBZ_DATABASES_0__CATALOG=source
+#export DBZ_DATABASES_0__USERNAME=postgres
+#export DBZ_DATABASES_0__PASSWORD=password
+
+# You can increase the index to config more database for the reader
+#export DBZ_DATABASES_1__TYPE=ORACLE
+#export DBZ_DATABASES_1__HOSTNAME=localhost
+#export DBZ_DATABASES_1__PORT=1521
+#export DBZ_DATABASES_1__CATALOG=ORCLCDB/ORCLPDB1
+#export DBZ_DATABASES_1__USERNAME=oracle
+#export DBZ_DATABASES_1__PASSWORD=password
+
+##########################################
+# Optional Parameters Below              #
+##########################################
+
+# Configure logging
+# Generic loglevel
+#export QUARKUS_LOG_LEVEL=DEBUG
+# Loglevel for a single package
+#export QUARKUS_LOG_CATEGORY__COM_ENTERPRISEDB__LEVEL=DEBUG
+
+# BigData max size after serialization. If bigger than this value, it will be sent to s3 (1Mb by default)
+# Should be lower than `max.message.bytes` (broker setting) to avoid issues.
+#export BIGDATA_MAX_BYTE_SIZE=1000000
+```
+
+## Running the reader
+
+```shell
+cd /opt/cdcreader
+./run-cdcreader.sh
+```
+
+### Parameters explanation
+
+#### DBZ_ID
+ID to identify the reader
```

Comment:
@yidian1997 can you add here some more info?
- Users can define it with the name they thought will be more usefull
- Are there any restrictions on the id length?
- are there any restrictions about the characters can be used?
- can 2 different reades have the same dbz_id?

Reply:
sure, I will think about it and update the doc

Reply:
I need do some test about these questions

Reply:
thanks for the update @yidian1997

Reply:
I update the constraints of DBZ_ID according to my test